repos/spack_repo/builtin/packages/root/package.py

```diff
@@ -895,11 +895,6 @@ def add_include_path(dep_name):
             # warnings when building against ROOT
             env.unset("MACOSX_DEPLOYMENT_TARGET")
 
-    @property
-    def root_library_path(self):
```

Comment:
This may be relied upon in downstream package repos. May need deprecation somehow?

FYI @jmcarcell

Reply:
Right, outside the main spack repo. Will restore.